.deploy/k9/prod-gcp-k9saksbehandling.json

```diff
@@ -0,0 +1,27 @@
+{
+  "environment": "prod",
+  "minReplicas": "2",
+  "maxReplicas": "6",
+  "limits": {
+    "cpu": "2",
+    "mem": "2048Mi"
+  },
+  "requests": {
+    "cpu": "500m",
+    "mem": "1024Mi"
+  },
+  "dbTier": "TODO",
```

Comment:
Fix TODO.

Reply:
Denne gjenstår før prodsetting @espenjv :)

Reply:
Ja, kva tier vi går for avhenger vel av kor mykje data vi kjem til å lagre? Veit du kvar man ser dei ulike alternativa?

Reply:
Det finnes en oversikt her: [https://cloud.google.com/sql/docs/mysql/instance-settings](https://cloud.google.com/sql/docs/mysql/instance-settings)